### ISSUE TITLE
Home: Page does not have a heading level 1

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#3]
**URL**: https://hcdev.openplus.ca/en
**User Agent:** Chrome/Firefox
**Language**: both

**Issue Description**
The page does not have heading `<h1>.` Headings should be in hierarchical order. H1->H2-H3 etc.

**Steps to reproduce:** Visual, source code

**Expected results:** All pages should have H1 and this page seems like "Recalls and safety alerts" is the name of the application so that is not H1. H1 and page title are generally the same and "skip to main content" link takes the user to H1.

**Comment by ndolter** [comment]
Validated in both French and English that H1 is now 'Find recalls, advisories and safety alerts'.  Skip to main content will navigate the user to this.
